packages/terra-clinical-item-display/src/ItemDisplay.jsx

```diff
@@ -115,27 +116,24 @@ const ItemDisplay = ({
     textWrapper = <strong>{text}</strong>;
   }
 
-  let ariaLabel;
+  let hiddenStyleMeaning;
+  let hiddenStyleMeaningEnd;
   if (textStyleMeaning) {
-    ariaLabel = `${textStyleMeaning}, ${text}, ${intl.formatMessage({ id: 'Terra.item-display.textStyleMeaningEnd' }, { textStyleMeaning })}`;
+    hiddenStyleMeaning = textStyleMeaning;
+    hiddenStyleMeaningEnd = intl.formatMessage({ id: 'Terra.item-display.textStyleMeaningEnd' }, { textStyleMeaning });
   } else if (textStyle === TextStyles.STRIKETHROUGH) {
-    ariaLabel = `${intl.formatMessage({ id: 'Terra.item-display.textStyleMeaningStrikethrough' })}, ${text}, ${intl.formatMessage({ id: 'Terra.item-display.textStyleMeaningStrikethroughEnd' })}`;
+    hiddenStyleMeaning = intl.formatMessage({ id: 'Terra.item-display.textStyleMeaningStrikethrough' });
+    hiddenStyleMeaningEnd = intl.formatMessage({ id: 'Terra.item-display.textStyleMeaningStrikethroughEnd' });
   }
 
   return (
     <div {...customProps} className={componentClassNames} aria-disabled={isDisabled}>
       {displayIcon}
-      {ariaLabel ? (
-        <span aria-label={ariaLabel}>
-          <div data-terra-clinical-item-display-text className={textClassNames} aria-hidden="true">
-            {textWrapper}
-          </div>
-        </span>
-      ) : (
-        <div data-terra-clinical-item-display-text className={textClassNames}>
-          {textWrapper}
-        </div>
-      )}
+      {hiddenStyleMeaning ? <VisuallyHiddenText text={hiddenStyleMeaning} /> : null}
```

Comment:
nit: I believe this works the same? Also maybe the same on line 136

```suggestion
      {hiddenStyleMeaning && <VisuallyHiddenText text={hiddenStyleMeaning} />}
```